### PR TITLE
Switch to mainnet

### PR DESCRIPTION
**Pull request by chrislatorres**

### CODE REVIEW

**Comment by chrislatorres** [comment]
Tested locally, this looks good to go

File: blockchain.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ function _setMainChain(isMainChain) {
   }
 }
 // _setMainChain(!/^test\./.test(location.hostname));
-_setMainChain(/^main\./.test(location.hostname));
+_setMainChain(true);
```
Comment (avaer) [inline blockchain.js:40]: Import the constant?

**Review by avaer — COMMENTED** [review]
One thing and this is ready to go, but we should deploy the contracts first.

File: blockchain.js (hunk 1 above)
Comment (chrislatorres) [reply blockchain.js]: added https://github.com/webaverse/app/pull/879/commits/0981f3fe49e92719fd2576cfa7eedc5e560c11b3

**Pull request merged**